$NetBSD: patch-at,v 1.14 2015/12/02 21:14:16 leot Exp $

Don't create directories, do that with OWN_DIRS and OWN_DIRS_PERMS

--- scheduler/Makefile.orig	2015-02-17 13:10:19.000000000 +0000
+++ scheduler/Makefile
@@ -145,6 +145,8 @@ install:	all install-data install-header
 #
 
 install-data:
+
+install-dirs:
 	echo Creating $(SERVERBIN)/driver...
 	$(INSTALL_DIR) -m 755 $(SERVERBIN)/driver
 	echo Creating $(SERVERROOT)...
